### PR TITLE
feature(dropdown-responsive-and-update-storybook): Upadate ProductLis…

### PR DESCRIPTION
**Pull request by patel-prakashkumar**
Hi, 
I have change CSS file and Update Storybook. Remove ProductListDropdown from Container folder 

# Description

Please provide a short summary explaining what this PR is about.

Fixes # (issue)

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have followed the name conventions for CSS Classnames and filenames, Components names and filenames, Style filenames, if you are in doubt check the the project README.MD and here https://github.com/HackYourFuture-CPH/curriculum/blob/master/review/review-checklist.md
- [ ] I have commented my code, particularly in hard-to-understand areas, if you code was simple enough mark the box anyway
- [ ] I have made corresponding changes to the documentation, if you code was simple enough mark the box anyway
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: packages/client/src/components/ProductListDropDown/ProductListDropDown.component.js [hunk 1]
```diff
@@ -1,24 +1,78 @@
...
+        <select
+          className="view-dropdown-select"
+          value={value}
+          onChange={handleChange}
+        >
+          {allFilter.map((optionAllFilter) => (
+            <option value={optionAllFilter}>{optionAllFilter}</option>
+          ))}
+        </select>
+      </div>
```
Comment (shpomp) [inline packages/client/src/components/ProductListDropDown/ProductListDropDown.component.js:69]: Those are 3 basically identical pieces of code.
Which in React means only one thing :) 
Make it into a component that gets what it needs through props ✅

**Review by shpomp — CHANGES_REQUESTED** [review]
Very nice! 👏🏽

File: packages/client/src/components/ProductListDropDown/ProductListDropDown.component.js (hunk 1 above)
Comment (patel-prakashkumar) [reply packages/client/src/components/ProductListDropDown/ProductListDropDown.component.js:69]: Thank you feedback. I have add separate components.

**Review by 27ManishaPatel — APPROVED** [review]

**Review by shpomp — APPROVED** [review]
Nice! 👏🏽  ⭐

**Pull request merged**